tools/data_manipulation/xarray/macros.xml

```diff
@@ -0,0 +1,4 @@
+<macros>
```

Comment:
you actually can put this into the same XML tool file.
https://github.com/galaxyproject/tools-iuc/blob/5fe46c36adac4843cd548802073aa62b0afc61cd/tools/bellerophon/bellerophon.xml#L3

Reply:
If you like, but no wait you are correct that should be shared across the other tools.

Reply:
I like `+galaxy@VERSION_SUFFIX@` (I did not really know what to do with `VERSION_SUFFIX`). Thanks!